README.md

```diff
@@ -1,3 +1,4 @@
-# SpectralNormalization
+# Spectral Normalization in Swift for TensorFlow
 
-A description of this package.
+## Environment
+- [Dockerfile](https://cloud.docker.com/u/crcrpar/repository/docker/crcrpar/swift-jupyter) build from [google/swift-jupyter](https://github.com/google/swift-jupyter) on 2019/07/14.
```

Comment:
TODO: Make this docker image public.

Reply:
Also, mention that I run this on `swift-tensorflow-DEVELOPMENT-2019-06-17-a.xctoolchain`.